File: trapp/scratch/check_json.js
const fs = require('fs');

try {
    const content = fs.readFileSync('results.json', 'utf8');
    const data = JSON.parse(content);
    console.log("JSON is valid. Number of items:", data.length);
} catch (e) {
    console.error("Error:", e.message);
    process.exit(1);
}

File: trapp20260601/scratch/check_json.py
import json
import sys

try:
    with open('results.json', 'r', encoding='utf-8') as f:
        data = json.load(f)
    print("JSON is valid. Number of items:", len(data))
except json.JSONDecodeError as e:
    print(f"JSONDecodeError: {e}")
    sys.exit(1)
except Exception as e:
    print(f"Error: {e}")
    sys.exit(1)
